### PR TITLE
Fixes documentation and Gives Proper Volto-Core Guide

### LINKED ISSUE #5552: Some Grammatical and Spelling errors in Documentation
1) In `Approachibility` section of `Design principles`.  , usage of word `ultimatelly `.
2) `Decoupling` section of same page `So let's them keep them decoupled` should be `So let's  keep them decoupled`.
3) In `Browser Compatibility` of `Language Feature` page  , `By doing this, enables the bundles` should be `By doing this, it 
     enables the bundles`.
4) In `About design systems` of `Style Guide` section:
     a) `It will evolve constantly with the product, the tools and the new technologies.` can be made more efficient as `It constantly 
          evolves with the product,tools and new technologies.`

     b)`Volto (and thus, Plone 6) has a design system in the making since some years ago.` can be made more efficient as `Volto (and thus, Plone 6) has a design system in the making for several years.`

### PR DESCRIPTION
**Pull request by ujjwaleee26**
Fixes #5552     
Also #5554

### CODE REVIEW

**Comment by ujjwaleee26** [comment]
Can I please get a review

**Comment by ujjwaleee26** [comment]
In my last PR also there was this `@plone/components` and `@plone/volto` which were showing expected. Like are these some kindof of unit test or so?

File: docs/source/contributing/developing-core.md
Comment (stevepiercy) [inline docs/source/contributing/developing-core.md:1]: Revert. We need to keep this file. Why did you delete it? We want to replace [this section](https://6.docs.plone.org/volto/contributing/index.html#install-volto-for-development) with a link to this file.

File: docs/source/contributing/index.md
Comment (stevepiercy) [inline docs/source/contributing/index.md:1]: Revert all changes to this file.

File: docs/source/contributing/language-features.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ Babel preset-env uses `browserlist` which gives the ability to micromanage the
 transformations needed by the current project depending of the browser support you are
 currently targeting.
 
-By doing this, enables the bundles to be smaller, as the resultant code does not need to
+By doing this, it enables the bundles to be smaller, as the resultant code does not need to
```
Comment (stevepiercy) [inline docs/source/contributing/language-features.md:39]:
```suggestion
By doing this, it enables the bundles to be smaller, as the resulting code does not need to
```

File: docs/source/contributing/style-guide.md [hunk 2]
```diff
@@ -11,9 +11,9 @@ myst:
 
 ## About design systems
 
-A Design System is the single source of truth which groups all the elements that will allow the teams to design, realize and develop a product. So a Design System is not a deliverable, but a set of deliverables. It will evolve constantly with the product, the tools and the new technologies.
+A Design System is the single source of truth which groups all the elements that will allow the teams to design, realize and develop a product. So a Design System is not a deliverable, but a set of deliverables. It constantly evolves with the product,tools and new technologies.
```
Comment (stevepiercy) [inline docs/source/contributing/style-guide.md:14]: Our style guides uses one sentence per line, although older docs such as this file did not follow this practice.

```suggestion
A Design System is the single source of truth which groups all the elements that will allow the teams to design, realize and develop a product.
So a Design System is not a deliverable, but a set of deliverables.
It constantly evolves with the product, tools, and new technologies.
```

File: docs/source/contributing/style-guide.md [hunk 3]
```diff
@@ -11,9 +11,9 @@ myst:
...
 
-Volto (and thus, Plone 6) has a design system in the making since some years ago. Albert Casado Celma (@albertcasado) has been working on it and providing with a lot of hints and advices about how Volto's UI/UX should be. As stated at the beginning, a design system is a continuous effort, that evolves over time with the users' feedback and needs.
+Volto (and thus, Plone 6) has a design system in the making for several years. Albert Casado Celma (@albertcasado) has been working on it and providing with a lot of hints and advices about how Volto's UI/UX should be. As stated at the beginning, a design system is a continuous effort, that evolves over time with the users' feedback and needs.
```
Comment (stevepiercy) [inline docs/source/contributing/style-guide.md:16]:
```suggestion
Volto (and thus, Plone 6) had a design system in the making for several years.
Albert Casado Celma (@albertcasado) had been working on it, and providing a lot of hints and advice about how Volto's UI/UX should be.
As stated at the beginning, a design system is a continuous effort, that evolves over time with the users' feedback and needs.
```

File: packages/volto/news/5552.bugfix
Comment (stevepiercy) [inline packages/volto/news/5552.bugfix:1]: Delete this file. It is not a bugfix because it is not code, but docs.

File: packages/volto/news/5554.documentation [hunk 4]
```diff
@@ -0,0 +1 @@
+Mereged the index.md file of Contributing to Volto and Develop Volto Core ,so that there is a complete flow present for developers and deleted to Develop Volto Core File.@ujjwaleee26
```
Comment (stevepiercy) [inline packages/volto/news/5554.documentation:1]: You have this inverted. Please see the issue #5556.

**Review by stevepiercy — CHANGES_REQUESTED** [review]
Sorry for the miscommunication. See #5556 with an improved description to describe what is needed.

File: docs/source/contributing/developing-core.md
Comment (ujjwaleee26) [reply docs/source/contributing/developing-core.md:1]: @stevepiercy , oh earlier you mentioned in issue description that transform index.md to develop volto core, so I thought maybe there was need of it, so sorry,I should have asked earlier

File: packages/volto/news/5554.documentation (hunk 4 above)
Comment (ujjwaleee26) [reply packages/volto/news/5554.documentation:1]: @stevepiercy  okay ,sorry for this PR, i am doing all the corrections, and instead of yarn commands portion in `index.md` is should just give the link of `developing volto core`.Right? ,cause anyway. yarn commands doesn't work with core -volto, only suitable for Volto handson training

**Comment by ujjwaleee26** [comment]
Sorry for the incoveninece @stevepiercy , but there is some problem in my vscode, even with `--no-verify`, I am not  able to push the changes.I will be closing this and raising a new one

**Comment by ujjwaleee26** [comment]
closing

File: packages/volto/news/5554.documentation (hunk 4 above)
Comment (stevepiercy) [reply packages/volto/news/5554.documentation:1]: Correct. Training is entirely separate from this issue.

**Comment by stevepiercy** [comment]
> Sorry for the incoveninece @stevepiercy , but there is some problem in my vscode, even with `--no-verify`, I am not able to push the changes.

Is your branch current with `main`?

**Comment by ujjwaleee26** [comment]
> > Sorry for the incoveninece @stevepiercy , but there is some problem in my vscode, even with `--no-verify`, I am not able to push the changes.
> 
> Is your branch current with `main`?

No  but , I figured out, there was corruption in my vscode, I reinstalled it, and I was able to push the changes
